### ISSUE TITLE
Getting handler._sign is not a function error

### ISSUE THREAD

**Issue by ankit-tripathi** [postmanlabs/newman#1484]
Hi All,

I am getting this error when ever I try to run my collection using Newman.
![image](https://user-images.githubusercontent.com/9364431/38190202-d8289398-3680-11e8-9a32-a4cd4268c052.png)

I have a very basic POST request which is used for log in to the application. When I run this using Postman, it runs fine, but gives error using Newman.
My collection:
![image](https://user-images.githubusercontent.com/9364431/38190276-287c5b90-3681-11e8-99e9-29f612a7c244.png)

NodeJs: 8.11.1
Newman: 3.8.2
Postman: 6.0.10

Please help!

**Comment by kunagpal** [comment]
@ankit-tripathi I was unable to reproduce this with Newman v3.9.3 on Node v8.11.1. Could you confirm that this issue persists with Newman v3.9.3? Also, could you run your collection as follows and share the logs? Thanks!

```
SET NODE_DEBUG=request
newman run sample.postman_collection.json
```

**Comment by ankit-tripathi** [comment]
Same error :(

![image](https://user-images.githubusercontent.com/9364431/38191449-63e05358-3686-11e8-87b7-4c428e987602.png)

**Comment by ankit-tripathi** [comment]
@kunagpal , I can't update the newman as I am working on VM having restricted internet access. 
Can you please help me with the procedure of updating Newman offline?

**Comment by vinaysuri1** [comment]
@kunagpal - scripts are working for me in NEWMAN but failing with this error in JENKINS.
What can be the reason for this issue? How can I resolve it?

**Comment by mtoothman** [comment]
I also experienced this error, and it was resolved by updating to 3.9.3. Thanks!
